### PR TITLE
New version fixes, WIP

### PR DESCRIPTION
**Pull request by ethanbb**

### CODE REVIEW

File: mdj-names3-test.js [hunk 1]
```diff
@@ -367,13 +366,13 @@ async function byNamesDuring (dates, browser) {
...
     assignmentData.currentIndex = nameIndex;
-    fs.writeFileSync(assignmentPath, JSON.stringify(assignmentData));
+    fs.writeFileSync(assignmentPath, JSON.stringify(assignmentData, null, 2));
 
   }  // ends while name index <= ending index
 
   // Record that this data was finished
   assignmentData.completed = true;
-  fs.writeFileSync(assignmentPath, JSON.stringify(assignmentData));
+  fs.writeFileSync(assignmentPath, JSON.stringify(assignmentData, null, 2));
```
Comment (ethanbb) [inline mdj-names3-test.js]: pretty-print with tab width of 2 spaces

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,6 @@
 node_modules/
-/data-cp/*/
-/data-mdj/*/
+/data-cp/*
+/data-mdj/*
```
[the remark this answers is not in the document]
Comment (ethanbb) [reply .gitignore]: OK got it. I had older files in the `data-mdj` directory which I didn't want to push, but I should just move those.

File: assignments/1m.json [hunk 3]
```diff
@@ -14,6 +14,6 @@
   "alerts": "yes",
   "note_done": "Needs to be implemented. Object of finished indexes",
   "done": {},
-  "note_done": "Needs to be implemented. Will allow redoing 'done' indexes",
+  "note_redo": "Needs to be implemented. Will allow redoing 'done' indexes",
```
[the remark this answers is not in the document]
Comment (ethanbb) [reply assignments/1m.json]: Should I get rid of this note then?

File: mdj-names3-test.js [hunk 4]
```diff
@@ -87,7 +86,7 @@ if (!assignmentID) {
   throw Error('I think you used the default assignemt ID (24z). That\'s not a real one.'.yellow);
 }
 
-const assignmentPath = 'assignemnts/' + assignmentID + '.json'
+const assignmentPath = './assignments/' + assignmentID + '.json'
```
[the remark this answers is not in the document]
Comment (ethanbb) [reply mdj-names3-test.js]: Yep, it's been working for me.

File: assignments/1m.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (knod) [reply assignments/1m.json]: Yeah, I think that's what's conflicting.

File: mdj-names3-test.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (knod) [reply mdj-names3-test.js]: That's really weird, but awesome. `require` should up its game!

File: assignments/1m.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ethanbb) [reply assignments/1m.json]: Oh I didn't realize there was a conflict, sorry!

File: data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt [hunk 5]
```diff
@@ -0,0 +1,194 @@
+1561924964288_MJ-05205-CR-0000418-2018_01_17_12_18_namei_3_page_1
```
Comment (ethanbb) [inline data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt]: Dammit, sorry, why isn't this being ignored?

Comment (knod) [reply data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt]: That's ok, that doesn't need to be ignored

Comment (knod) [reply data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt]: I think.

File: data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+1561924969580_MJ-10208-CR-0000245-2018_01_17_12_18_namei_3_page_1
+1561924974103_MJ-10104-CR-0000188-2018_01_17_12_18_namei_3_page_1
+1561924977502_MJ-44301-CR-0000039-2018_01_17_12_18_namei_3_page_11561925834837_MJ-32248-CR-0000456-2018_01_17_12_18_namei_10_page_1
```
Comment (knod) [inline data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt]: Do you know what happened here?

Comment (ethanbb) [reply data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt]: Yeah, I fixed it. The above lines were from before I changed it to add a newline after each entry, and I forgot the newline after the last one when manually fixing it.

**Pull request merged**